### PR TITLE
Create Progress component

### PR DESCRIPTION
**Pull request by JjuliaP**
## JIRA

* [Main JIRA ticket](http://ssu-jira.softserveinc.com/browse/LVPYTP-50)

## Code reviewers

- [ ] @dmytrorevak
- [ ] @IvanAndrukh
- [ ] @JjuliaP
- [ ] @makarman
- [ ] @noobhunters
- [ ] @Silent777
- [ ] @Staryj812

### Second Level Review

- [ ] @ssOleg
- [ ] @lhalam

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: eventually/static/src/containers/progress/Progress.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import LinearProgress from 'material-ui/LinearProgress';
+
+export const lightGreen500 = '#8bc34a';
```
Comment (redmyt) [inline eventually/static/src/containers/progress/Progress.js]: for what we export this constant?

Comment (redmyt) [reply eventually/static/src/containers/progress/Progress.js]: we can import all colors we need from material-ui
for example `import {blue500, yellow600, lime500} from 'material-ui/styles/colors';`

File: eventually/static/src/containers/progress/Progress.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+const style={
```
Comment (redmyt) [inline eventually/static/src/containers/progress/Progress.js]: please add whitespaces at both sides near to equal char =)

File: eventually/static/src/mainRouter.js [hunk 3]
```diff
@@ -4,12 +4,14 @@ import Home from './containers/Home';
 import Profile from './containers/profile/Profile';
```
Comment (redmyt) [inline eventually/static/src/mainRouter.js]: do we need this component in the containers and router?
maybe it makes sense to move it to components and we would use that component in varies pages.

File: eventually/static/src/containers/progress/Progress.js (hunk 1 above)
Comment (JjuliaP) [reply eventually/static/src/containers/progress/Progress.js]: done

File: eventually/static/src/containers/progress/Progress.js (hunk 2 above)
Comment (JjuliaP) [reply eventually/static/src/containers/progress/Progress.js]: done

File: eventually/static/src/mainRouter.js (hunk 3 above)
Comment (JjuliaP) [reply eventually/static/src/mainRouter.js]: done

**Review by redmyt — APPROVED** [review]

**Pull request merged**